src/functionalTest/resources/scenarios/RIA-7338-update-sign-language-appellant-flag.json

```diff
@@ -0,0 +1,160 @@
+{
+  "description": "RIA-7338 update sign language choice - legal rep",
+  "enabled": "false",
```

Comment:
Should this test be enabled by $featureFlag.isSubmitHearingRequirementsEnabled ?